src/superphot_plus/utils.py

```diff
@@ -169,32 +177,35 @@ def flux_model(cube, t_data, b_data):
         A / (1.0 + np.exp(-phase[phase < gamma] / tau_rise)) * (1.0 - beta * phase[phase < gamma])
     )
 
-    # for secondary band
-    start_idx = 7
-    A_b = A * cube[start_idx]
-    beta_b = beta * cube[start_idx + 1]
-    gamma_b = gamma * cube[start_idx + 2]
-    t0_b = t0 * cube[start_idx + 3]
-    tau_rise_b = tau_rise * cube[start_idx + 4]
-    tau_fall_b = tau_fall * cube[start_idx + 5]
-
-    inc_band_ix = np.array(b_data) == "g"
-    phase_b = (t_data - t0_b)[inc_band_ix]
-    phase_b2 = (t_data - t0_b)[inc_band_ix & (t_data - t0_b < gamma_b)]
-
-    f_model[inc_band_ix] = (
-        A_b
-        / (1.0 + np.exp(-phase_b / tau_rise_b))
-        * (1.0 - beta_b * gamma_b)
-        * np.exp((gamma_b - phase_b) / tau_fall_b)
-    )
-    f_model[inc_band_ix & (t_data - t0_b < gamma_b)] = (
-        A_b / (1.0 + np.exp(-phase_b2 / tau_rise_b)) * (1.0 - phase_b2 * beta_b)
-    )
+    for e, ub in enumerate(ordered_bands):
```

Comment:
The variable names `e` and `ub` are a bit difficult to follow. I'm guessing they stand for something that I can't figure out.

What about something like `band_num` and `band_val`?